README.md

```diff
@@ -426,6 +445,57 @@ patchModules.config = [
 ]
 ``` 
 
+Compilation
+===
+
+Compilation to a specific Java release
+----
+
+For Java releases 6-8, see [Separate compilation of `module-info.java`](#separate-compilation-of-module-infojava).
+
+For Java releases 9+, this plugin provides an easy way to set the `--release` option of the `compileJava` task
```

Comment:
Maybe add a link to some docs for the --release option. E.g. http://openjdk.java.net/jeps/247

Reply:
Yes, a very good idea!